modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc

```diff
@@ -3,150 +3,63 @@
 
 == Introduction
 
-Federated Cloud Sharing is now managed by the Federation app (9.0+), and
-is now called Federation sharing. When you enable the Federation app you
-can easily and securely link file shares between ownCloud servers, in
-effect creating a "cloud" of ownCloud installations.
+Federated Cloud Sharing is managed by the Federation app. When you enable the Federation app you can easily and securely link file shares between ownCloud servers, in effect creating a "cloud" of ownCloud installations.
 
 [IMPORTANT]
 ====
 For security reasons federated sharing **strictly requires HTTPS (SSL/TLS)**.
 ====
 
-[[sharing-with-owncloud-8-and-older]]
-== Sharing With ownCloud 8 and Older
+== Configuration
+Follow these steps to establish a trusted connection between two servers.
 
-Direct Federation shares (label-direct-share-link) are not supported in
-ownCloud 8 and older, so you must create Federation shares with public
-links (label-public-link-share).
-
-[[creating-a-new-federation-share-9.0-only]]
-== Creating a new Federation Share (9.0+ only)
-
-Follow these steps to create a new Federation share between two ownCloud
-9.0+ servers. This requires no action by the user on the remote server;
-all it takes is a few steps on the originating server.
-
-1.  Enable the Federation app.
-2.  Go to your ownCloud Admin page and scroll to the Sharing section.
-Verify that *Allow users on this server to send shares to other servers*
-and *Allow users on this server to receive shares from other servers* are enabled.
-3.  Now go to the Federation section. By default, *Add server
-automatically once a federated share was created successfully* is
-checked. The Federation app supports creating a list of trusted ownCloud
-servers, which allows the trusted servers to exchange user directories
-and auto-complete the names of external users when you create shares. If
-you do not want this enabled, then un-check it.
-+
-image:configuration/files/federation-0.png[image]
-4. Then, go to your Files page and select a folder to share. Click the
-share icon, and then enter the username and URL of the user on the
-remote ownCloud server. In this example, that is
-`freda@https://example.com/owncloud`. When ownCloud verifies the link,
-it displays it with the *(remote)* label. Click on this label to establish the link.
-+
-image:configuration/files/federation-2.png[image]
-5. When the link is successfully completed, you have a single share
-option, and that is *can edit*.
+. Verify valid SSL certificates on both servers.
+. Verify your `overwrite.cli.url` is configured to the correct URL and instead of `localhost`.
+. Reset your `oc_jobs` table.
```

Comment:
Why is this needed?

Reply:
Because it's clean. We don't know if the federation cron job already run or not. Setting everything to 0 at the start creates good conditions for the federation to work.

Reply:
Do this only for the federation background job not for all jobs.

Reply:
@pako81, which class column value do you filter on to only clean out federation background jobs?

Reply:
class is `OCA\Federation\SyncJob`

Reply:
filtered